tests/modules/restful_test/restful_test.install

```diff
@@ -54,5 +54,37 @@ function restful_test_schema() {
     'primary key' => array('pid'),
   );
 
+  // Defining table via hook_install() due to drupal_write_record().
```

Comment:
Correct.  `drupal_write_record()` uses ` drupal_get_schema()` to find the names of the fields for the supplied table, and if the table is not defined in the schema, it will return `FALSE`.